scripts/kurtosis_config.yaml

```diff
@@ -26,8 +26,10 @@ mev_params:
   # Bolt-specific images:
   # Adding the `bolt_boost_image` will start the devnet with Bolt-Boost
   # instead of MEV-Boost by Flashbots
-  bolt_boost_image: ghcr.io/chainbound/bolt-boost:0.1.0
+  # bolt_boost_image: ghcr.io/chainbound/bolt-boost:0.1.0
   bolt_sidecar_image: ghcr.io/chainbound/bolt-sidecar:0.1.0
+  # bolt_sidecar_extra_params:
+  #   firewall_rpcs: "ws://host.docker.internal:8015/api/v1/firewall_stream"
```

Comment:
Do you still want this to be commented out?

Reply:
Yes, the firewall-ed setup should not be the default because it requires a bolt-rpc ready to run, which isn't publicly available as of now.
I think it's still useful to leave this commented so that's easy to turn it on if needed, like it has been done between MEV-Boost fork and bolt-boost